apps/organization/models.py

```diff
@@ -69,3 +69,8 @@ def __str__(self):
         return f'{self.pk} : ({self.short_name}) {self.title} ' + (
             '(MERGED)' if self.parent else ''
         )
+
+    def get_organization_type_display(self):
+        if self.parent:
+            return self.parent.organization_type.title
+        return self.organization_type.title
```

Comment:
Let's use `self.data` for parent/self reference.
```suggestion
    def get_organization_type_display(self):
        return self.data.organization_type.title
```